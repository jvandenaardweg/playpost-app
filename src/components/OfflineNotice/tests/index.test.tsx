import React from 'react';
import { render, RenderAPI } from 'react-native-testing-library';

import { OfflineNotice } from '../index';

describe('OfflineNotice', () => {

  describe('rendering', () => {
    let wrapper: RenderAPI;

    beforeAll(() => {
      wrapper = render(<OfflineNotice />);
    });

    it('should render correctly', () => {
      expect(wrapper.toJSON()).toMatchSnapshot();
    });
  });
});
