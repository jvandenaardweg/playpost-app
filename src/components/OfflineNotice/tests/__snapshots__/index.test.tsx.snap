// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OfflineNotice rendering should render correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#b52424",
      "height": 30,
      "justifyContent": "center",
      "width": "100%",
    }
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "#000000",
          "fontFamily": "Inter-SemiBold",
          "fontSize": 13,
          "lineHeight": 16,
        },
        Object {
          "color": "#ffffff",
        },
      ]
    }
  >
    No internet connection, playing is limited
  </Text>
</View>
`;
